habitat_code/example-big-sur.py

```diff
@@ -0,0 +1,117 @@
+# %% [markdown]
+# # Changes in vegetation caused by the Mud Creek Landslide
+# 
+# ![Mud Creek Landslide](https://d9-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/styles/list_item/public/thumbnails/image/BigSur1_May20SlideDES_0.jpg)
+# 
+# > Image source: [USGS](https://www.usgs.gov/special-topics/big-sur-landslides/science/mud-creek-landslide-may-20-2017)
+
+# %%
+import re
+
+import earthaccess
```

Comment:
I don't have the earthaccess library. Please include environment.yml or requirements.txt and instructions for installing necessary libraries